src/Icon/_variables.scss

```diff
@@ -0,0 +1,7 @@
+
+// Icons sizes
+
+$icon-inline: .9rem !default;
```

Comment:
```suggestion
$icon-inline: .8em !default;
```
This way inline icons will size in relation to font size.

Reply:
@abutterworth fixed!